content/includes/header.html

```diff
@@ -34,6 +34,9 @@
 <meta content="{{ tag }}" property="article:tag">
 {% endfor %}
 {% endif %}
+{% if title == 'Education Benefits Application Process' or title == 'Education Benefits Eligibility' %}
+<link rel="import" href="/education/wizard/index.html">
```

Comment:
This would be really nice, but it doesn't look like it's possible to render liquid within [collections](https://github.com/jekyll/jekyll/issues/6211).